src/_P036_FrameOLED.ino

```diff
@@ -761,13 +726,18 @@ boolean Plugin_036(uint8_t function, struct EventStruct *event, String& string)
             }
 
             nextFrameToDisplay = LineNo / ScrollingPages.linesPerFrame; // next frame shows the new content
+            if (nextFrameToDisplay == NFrames) {   // corner-case?
+              nextFrameToDisplay--;
+            }
 
             bNoDisplayOnReceivedText = getBitFromUL(PCONFIG_LONG(0), 18);  // Bit 18 NoDisplayOnReceivedText
             if (UserVar[event->BaseVarIndex] == 0 && !bNoDisplayOnReceivedText) {
               // display was OFF, turn it ON
-              displayTimer = PCONFIG(4);
               display->displayOn();
               UserVar[event->BaseVarIndex] = 1;      //  Save the fact that the display is now ON
+              sendData(event);                       // Improvement to immediately show the page the text is placed on
```

Comment:
How can the Framed display task send data?
It has no controllers connected.

Reply:
Well, I wasn't sure I needed this call, but it seemed to work better with this call in. I'll re-test.

Reply:
That sendData, and another occurrence, wasn't needed, removed.